src/auth/ui/vue/root.vue

```diff
@@ -221,16 +229,10 @@ export default defineComponent({
             })
             return [...unlocked, ...locked]
         },
-        authFormDisplayName() {
-            if (this.successfulAuthConnection === undefined) {
-                return ''
-            }
-            return AuthFormDisplayName[this.successfulAuthConnection]
-        },
     },
     methods: {
         isLandscape() {
-            return this.currWindowWidth > 1200
+            return this.currWindowWidth > 1120
```

Comment:
I think usually you would check this using viewport ratios and/or orientation. But I'm sure this works fine.